scripts/ngless-cwl/cwl/workflow-example.cwl

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env cwl-runner
+# This tool description was generated automatically by argparse2tool ver. 0.4.3-2
+# To generate again: $ ./ngless-trim.py -b ./ngless-trim.py --generate_cwl_tool
+# Help: $ ./ngless --help_arg2cwl
+
+cwlVersion: "cwl:v1.0"
+
+requirements:
+- $import: ngl-types.yml
+
+class: Workflow
+inputs:
+  start: File
+  trim_output_name: string
+  trim_method: ngl-types.yml#trim_method
+  trim_qual: int
+  trim_discard: int
+  map_output_name: string
+  stats_output_name: string
+  ref: File
```

Comment:
FYI, adding `doc` and `label` will make the workflow visualization prettier 

https://view.commonwl.org/workflows/github.com/Unode/ngless/tree/cwl_wrappers/scripts/ngless-cwl/cwl/workflow-example.cwl

Also will show up in the GUI for your CWL tools on various platforms if they are added there